judge/models/problem.py

```diff
@@ -23,6 +24,12 @@
            'License', 'Solution', 'TranslatedProblemQuerySet', 'TranslatedProblemForeignKeyQuerySet']
 
 
+def DisallowCharactersValidator(text):
```

Comment:
Final nit: perhaps `DisallowedCharactersValidator` is a better name, to match `DISALLOWED_CHARACTERS`?

Though, I guess this is really `disallowed_characters_validator` -- the casing misled me into thinking this was a class-based validator, which it isn't.

Reply:
fixed